### ISSUE TITLE
Send message in given topic

### ISSUE THREAD

**Issue by MPSebastian** [amosproj/amos2023ss01-apache-pulsar-ui#22]
## User story
1. As a user
2. I want to be able to add new messages to a given topic
3. So that I can produce new content for consumers.

## Acceptance criteria
* [ ] User has to provide topic + message
* [ ] Message is added to topic on pulsar
* [ ] Message can be retrieved later on

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
